### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by tonypartridge** [joomlatools/joomla-vagrant#183] [General]
Hi Guys, seeing a lot of issues with no replies and no commits since early last year... is it dead?

**Comment by johanjanssens** [comment]
Hello Tony,

Project is certainly not death, last major release was early 2020 and we are still using it actively for our day to day development. We are however also working on a Docker container to replace the Vagrant box so we are not planning any future feature releases anymore for the Vagrant box. 

With that said, as far as I'm aware there is only an open issue regarding Composer 2, see: https://github.com/joomlatools/joomlatools-vagrant/issues/178 There is a workaround for it by using Composer 1 for now. 

If there are any other issues you encounter let us know. Always happy to see if we can help.

**Comment by johanjanssens** [comment]
Hello @tonypartridge,

Little update from my end. Been working hard on new project called [Joomlatools Server](https://github.com/joomlatools/joomlatools-server) to replace the Vagrant box. Joomlatools Server uses Docker instead of Virtualbox/Vagrant. 

Other updates:

- **Joomlatools Composer**: We have already updated our composer installer to support Composer v2 this resolves https://github.com/joomlatools/joomlatools-vagrant/issues/178 

- **Joomlatools Console**: We are now working on [Joomlatools Console v2](https://github.com/joomlatools/joomlatools-console/milestone/20), adding more flexibility and improved support for Joomlatools Server.

Your feedback is most welcome, patches, contributions and PR's even more!

**Reply by tonypartridge** [reply]
Ohhh yes!!! I've enjoyed the vagrant for all apps as it's easy. But even happier with Docker! I'll have a play

**Reply by johanjanssens** [reply]
Thanks, some more info, Joomlatools Server is not your regular multi-container approach, I build a single container multi-process LAMP stack, supervised by [S6 overlay](https://github.com/just-containers/s6-overlay) and with some extra goodies like [Swoole](https://www.swoole.co.uk/). 

This multi-process approach allows to run the server locally with Docker but also run it with Gitpod and deploy it very easily on any cloud hosting that supports Docker. My personal favorite is https://fly.io/

Additionally the server can be used as a base for any LAMP app, and the benefit of Docker is that you can fully customize it further to your own needs, adding php extensions, linux binaries, tools, scripts etc. 

Here is Joomlatools Server deployed on fly.io: https://quiet-thunder-8745.fly.dev/  and go here to boot up a Gitpod: https://gitpod.io/#https://github.com/joomlatools/joomlatools-server
